schema.sql

```diff
@@ -33,4 +33,27 @@ ALTER TABLE animals
 ALTER TABLE animals 
     ADD foreign key (species_id) references species(id);
 ALTER TABLE animals 
-    ADD foreign key (owner_id) references owners(id);
\ No newline at end of file
+    ADD foreign key (owner_id) references owners(id);
+
+-- create vets table to the database
+CREATE TABLE vets (
+    id bigserial PRIMARY KEY,
+    name VARCHAR(100),
+    age INT,
+    date_of_graduation DATE
+);
+
+-- create specializations join table to the database
+CREATE TABLE specializations (
+    id bigserial PRIMARY KEY,
+    species_id INT REFERENCES species(id),
+    vet_id INT REFERENCES vets(id)
+);
+
+-- create visits table to join animals table and vets table
+CREATE TABLE visits (
+    id bigserial PRIMARY KEY,
```

Comment:
- [ ] `visits` table references the primary key of `animals` and `vets` table. So just the foreign keys are necessary. Kindly delete the `id PRIMARY KEY` .